QUARKUS-343.md

```diff
@@ -0,0 +1,48 @@
+# QUARKUS-343 - Quarkus and non JSON payload in REST endpoints
+
+## Scope of the testing
+The test scope includes validating Quarkus's capability to handle non-JSON payloads (XML, YAML, and images) efficiently in REST endpoints.
+Tests will cover both JVM and Native mode executions.
+#### Test scenarios
+
+        1. XML Payload Scenario:
+            * Verify the endpoint's ability to consume and produce XML payloads.
+            * Send a request with an XML payload to the endpoint.
+            * Validate the response, ensuring it matches the expected XML structure.
+    
+        2. YAML Payload Scenario:
+            * Check the endpoint's ability to return YAML from a YAML file.
+            * Send a request with a YAML payload from a file to an endpoint.
```

Comment:
I don't understand your point. You are sending REST HTTP requests with YAML payload, where you get the payload can only be relevant if
1. you test client feature that loads file itself (you dont')
2. you want to know nothing timeouts inside REST endpoint when you are loading stuff etc. that's fine.

It is absolutely not interesting where you get it, point is in whether you serialize it yourself, what content type you send and whether Quarkus can serialize / deserialize it for your. We test Quarkus, not loading of files inside unit tests.